config/default.yml

```diff
@@ -122,6 +122,12 @@ AssetSizeCSS:
   threshold_in_bytes: 100_000
   ignore: []
 
+AssetSizeCssStylesheetTag:
+  enabled: false
+  ignore: []
+  threshold_in_bytes: 100_000
+  ignore: []
```

Comment:
Looks like this was a mistake. The ignore: [] twice is redundant. You only need it once (in all the places above and here as well).